pay-api/src/pay_api/services/payment_service.py

```diff
@@ -180,23 +180,16 @@ def update_invoice(cls, invoice_id: int, payment_request: Tuple[Dict[str, Any]],
         invoice: Invoice = Invoice.find_by_id(invoice_id, skip_auth_check=False)
         # If the call is to apply credit, apply credit and release records.
         if is_apply_credit:
-            credit_balance = Decimal('0')
             payment_account: PaymentAccount = PaymentAccount.find_by_id(invoice.payment_account_id)
-            invoice_balance = invoice.total - (invoice.paid or 0)
-            if (payment_account.credit or 0) >= invoice_balance:
-                pay_service: PaymentSystemService = PaymentSystemFactory.create_from_payment_method(
-                    invoice.payment_method_code)
-                # Only release records, as the actual status change should happen during reconciliation.
-                pay_service.apply_credit(invoice)
-                credit_balance = payment_account.credit - invoice_balance
-                invoice.paid = invoice.total
-                invoice.save()
-            elif (payment_account.credit or 0) <= invoice_balance:
-                invoice.paid = (invoice.paid or 0) + (payment_account.credit or 0)
-                invoice.save()
-
-            payment_account.credit = credit_balance
-            payment_account.save()
+
```

Comment:
I don't think this is needed, I don't think we call `update_invoice` for EFT in the frontend at all? 
I'll have to look a little closer